doc/contributing/strategic-initiatives.md

```diff
@@ -7,15 +7,16 @@ agenda to ensure they are active and have the support they need.
 ## Current initiatives
 
 | Initiative             | Champion                    | Links                                         |
-| ---------------------- | --------------------------- | --------------------------------------------- |
-| Core Promise APIs      | [Antoine du Hamel][aduh95]  | <https://github.com/nodejs/TSC/issues/1094>   |
-| QUIC / HTTP3           | [James M Snell][jasnell]    | <https://github.com/nodejs/quic>              |
-| Shadow Realm           | [Chengzhong Wu][legendecas] | <https://github.com/nodejs/node/issues/42528> |
-| Startup Snapshot       | [Joyee Cheung][joyeecheung] | <https://github.com/nodejs/node/issues/35711> |
-| V8 Currency            | [Michaël Zasso][targos]     |                                               |
-| Next-10                | [Michael Dawson][mhdawson]  | <https://github.com/nodejs/next-10>           |
-| Single executable apps | [Darshan Sen][RaisinTen]    | <https://github.com/nodejs/node/issues/43432> |
-| Performance            |                             | <https://github.com/nodejs/performance>       |
+| ---------------------- | ------------------------------- | --------------------------------------------- |
+| Core Promise APIs      | [Antoine du Hamel][aduh95]      | <https://github.com/nodejs/TSC/issues/1094>   |
+| QUIC / HTTP3           | [James M Snell][jasnell]        | <https://github.com/nodejs/quic>              |
+| Shadow Realm           | [Chengzhong Wu][legendecas]     | <https://github.com/nodejs/node/issues/42528> |
+| Startup Snapshot       | [Joyee Cheung][joyeecheung]     | <https://github.com/nodejs/node/issues/35711> |
+| V8 Currency            | [Michaël Zasso][targos]         |                                               |
+| Next-10                | [Michael Dawson][mhdawson]      | <https://github.com/nodejs/next-10>           |
+| Single executable apps | [Darshan Sen][RaisinTen]        | <https://github.com/nodejs/node/issues/43432> |
+| Performance            |                                 | <https://github.com/nodejs/performance>       |
+| Primordials            | [Benjamin Gruenbaum][benjamingr]| <https://github.com/nodejs/performance>       |
```

Comment:
The link will need to be changed.

Reply:
```suggestion
| Primordials            | [Benjamin Gruenbaum][benjamingr]| <https://github.com/nodejs/TSC/issues/1439>   |
```

Reply:
Maybe that should be a tracking issue in core, unless we want to put `Refs: some-TSC-issue` in the commit messages instead..

Reply:
I thought I would replace this with the repository once it's created

Reply:
You mean before this PR lands or after?

Reply:
Before and i'll amend it, ideally

Reply:
updated